cli/crowdin.ts

```diff
@@ -135,95 +135,89 @@ function uploadBundledTranslationsAsync(crowdinDir: string, branch: string, prj:
     return nextFileAsync();
 }
 
-export function downloadTargetTranslationsAsync(parsed?: commandParser.ParsedCommand) {
-    const name = (parsed && parsed.args[0]) || "";
-    const crowdinDir = pxt.appTarget.id;
-    return crowdinCredentialsAsync()
-        .then(cred => {
-            if (!cred) return Promise.resolve();
+export async function downloadTargetTranslationsAsync(parsed?: commandParser.ParsedCommand) {
+    const name = parsed?.args[0];
```

Comment:
keeping this as `undefined` instead of `""`; we did a check against falsey values when using this anyway, no reason to coerce to string